.github/CONTRIBUTING.md

```diff
@@ -78,7 +78,31 @@ The following guidelines may answer most of your questions and help you get star
         cp guides/v2.2/install-gde/composer.md guides/v2.3/install-gde/composer.md
         ```
 
-- Make sure that you update the correct version(s) of documentation (v2.2, v2.3). If you are not sure what directory to put your content in, just do your best. We can help re-locate it (if necessary) during the review process.
+    - When you create a new topic (create a new `.md` file) and its content is the same for 2.2 and 2.3 versions, create a symbolic link.
```

Comment:
Move the section as noted in the image as a section entitled "Symbolic links" in the Tips for writing content section (after Edit metadata). I think it warrants its own section so people can easily see this information. Don't you?

Reply:
![Screen Shot 2019-09-11 at 8 15 24 AM](https://user-images.githubusercontent.com/14031342/64700566-c0571600-d46c-11e9-9aac-f024523a6b2a.png)

Reply:
of course, I do @shrielenee

**done**